### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by ArnaudRinquin**
Detailing:

* Installation (boring but must-have)
* Middleware setup
* `handle()` usage
* Event hooks, with a `sendAnalytics` use case

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,3 +136,126 @@ export function loadFoo(id) {
...
+## How to
+
+
+### Install
+
+The first step is to add `redux-pack` in your project
+
+```sh
+npm install -S redux-pack
+
```
Comment (lelandrichardson) [inline README.md]: missing some closing `\`\`\`` here

Comment (lelandrichardson) [inline README.md]: er, i guess you're making this one big code block. nvm.

**Comment by lelandrichardson** [comment]
Wow. Awesome! Thanks for doing this!

**Comment by ArnaudRinquin** [comment]
There are probably many errors or things to improve but it gives something to start with ;)

**Pull request merged**